usr/lib/cca_stdll/cca_specific.c

```diff
@@ -2609,6 +2609,29 @@ CK_RV cca_load_config_file(STDLL_TokData_t *tokdata, char *conf_name)
 }
 
 #ifndef NO_PKEY
+static CK_BBOOL ccatok_pkey_token_is_encr_with_new_mk(STDLL_TokData_t *tokdata,
```

Comment:
No need for these 2 functions, see other comments.

Reply:
Done.